src/components/banner/banner.module.css

```diff
@@ -36,3 +36,8 @@
 .warning .children {
   color: var(--figma-color-text-onwarning);
 }
+
+/* Link component does not have enough contrast on our warning bg */
+.warning :global(a) {
+  color: #394360;
```

Comment:
Hummm I think this won't work for anchor outside of a warning banner

Reply:
![image](https://user-images.githubusercontent.com/1063530/192264864-673ed028-7af7-446d-935a-51d86977b246.png)

Reply:
Forget it I miss the warning prefix